README.md

```diff
@@ -28,6 +28,8 @@ There are a few dependencies which needs to be downloaded and installed before y
 - [Git](https://git-scm.com/downloads)
 - [CMake](https://cmake.org/download) and [Ninja](https://github.com/ninja-build/ninja/releases) for __Linux or MacOS users__. For Windows users, it is part of the onboarding configuration tools intall.
 
+> Please note that this extension __only supports__ the release versions of ESP-IDF, you can still use the extension on `master` branch or some other branch, but certain feature might not work fully.
+
```

Comment:
I would suggest a link to [ESP-IDF support policy](https://github.com/espressif/esp-idf/blob/master/SUPPORT_POLICY.md) here to make sure people understand what we mean to _supported_ versions.